package.json

```diff
@@ -28,8 +28,7 @@
   },
   "homepage": "https://github.com/pinojs/pino-pretty#readme",
   "precommit": [
-    "lint",
-    "test"
+    "lint"
```

Comment:
please revert this change

Reply:
I will revert as soon as I fix the tests, currently the coverage is under 100% and it's failing to commit.
I wouldn't merge this PR until the tests are with 100% coverage anyway